lib/rubocop/config_formatter.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'yaml'
+
+module RuboCop
+  # Builds YAML config file from two hashes
+  class ConfigFormatter
+    NAMESPACES =
+      /^(Bundler|Gemspec|Layout|Lint|Metrics|
+         Naming|Performance|Rails|Security|Style)/x.freeze
+
+    def initialize(config, descriptions)
+      @config = config
+      @descriptions = descriptions
+    end
+
+    def dump
+      YAML.dump(unified_config).gsub(NAMESPACES, "\n\\1")
+    end
+
+    private
+
+    def unified_config
+      cops.each_with_object(config.dup) do |cop, unified|
+        next if cop == 'Rails'
```

Comment:
a comment why Rails was excluded would be nice

Reply:
Addressed this by refactoring in 8fb2d70.